command/ibft/propose/ibft_propose.go

```diff
@@ -32,6 +32,13 @@ func setFlags(cmd *cobra.Command) {
 		"the address of the account to be voted for",
 	)
 
+	cmd.Flags().StringVar(
+		&params.rawBLSPublicKey,
+		blsFlag,
+		"",
+		"the BLS PublicKey of the account to be voted for",
```

Comment:
Typo here, should be `public key`

Reply:
Fixed at a916d9cf5a0e4b06e00341c47c2bf231a68415ef